Add README and LICENSE files (#11)
Discordia-Development/wiki: README.md

[Discordia]: https://discordia.me
[Discord]: https://discord.com

[Easter Eggs]: https://discordia.me/en/easter-eggs
[User Settings]: https://discordia.me/en/user-settings

[Wiki.js v2]: https://wiki.js.org
[guide]: https://docs.requarks.io/en/editors/markdown

[server]: https://discord.gg/ZRJ9Ghh

[license]: https://github.com/Discordia-Development/wiki/blob/new/license.md

# Discordia

> [Discordia]( is of if not the largest Wiki all about [Discord].

Reviewer: ```suggestion
[Discordia] is a community-made wiki about [Discord].
```

> From [Easter Eggs] to [User Settings] do we cover (almost) all importand information you may want to know about Discord.

Reviewer: ```suggestion
From [Easter Eggs] to [User Settings] we cover (almost) any information you may want to know about Discord.
```

## Wanna help?
Great! We appreciate any help to expand and improve the wiki.  

> You can contribute to this wiki in two ways.

Reviewer: ```suggestion
You can contribute to our project in a few ways.
```

### Markdown Editor
We use [Wiki.js v2] for rendering and handling the wiki pages on our side and recommend you to use their Markdown Editor for editing and creating pages.  
Please take a look at their [User Guide][guide] to get a basic understanding of what the editor may offer.

### Pull Request
You can also create a Pull request here on GitHub to update and/or add pages of the wiki.  
Please make sure that your PR targets the `new` branch, as it is the main one used by the wiki itself.

## Questions?
If you have any questions about us or the wiki, don't hesitate to join our [Discord Server][server] for help.

## Licenses

> If not stated otherwhise are the contents displayed on the wiki licensed under [**Creative Commons Attribution Share Alike 4.0 International**][license].  

Reviewer: ```suggestion
Unless stated otherwise, the content displayed on the wiki are licensed under the [**Creative Commons Attribution Share Alike 4.0 International**][license].  
```

> Any images depicted on the wiki are the intelectual property of their respective copyright holder.

Reviewer: ```suggestion
All images and information covered by the Wiki are the intellectual property of Discord, inc. We are not endorsed or compensated by Discord for any information provided, and content may be amended or removed at any time to comply with applicable copyright law. The information provided is accurate to the best of our human ability, but we don't guarantee that it's absolutely correct, and take no responsibility for how the information is used.
```

Author: I thought about also adding a line about our goal being to use the material in a fair use way, to provide information in a simple to understand way, and that the copyrighted material is therefore used to support this goal.
Is that an idea?
